docs/sdks/uid2-sdk-ref-cplusplus.md

```diff
@@ -76,6 +76,35 @@ Available information returned through the SDK is outlined in the following tabl
 | `KeysNotSynced` | The client has failed to synchronize keys from the UID2 service. |
 | `VersionNotSupported` |  The client library does not support the version of the encrypted token. |
 
+
+## Usage for UID2 Sharers
+
+A UID2 sharer is any participant that wants to share UID2s with another participant. Raw UID2s must be encrypted into UID2 tokens before sending them to another participant. For an example of usage, see [com.uid2.client.test.IntegrationExamples](https://github.com/IABTechLab/uid2-client-java/blob/master/src/test/java/com/uid2/client/test/IntegrationExamples.java) (`runSharingExample` method).
+
+The following instructions provide an example of how you can implement sharing using the UID2 SDK for Java, either as a sender or a receiver.
+
+1. Create an IUID2Client reference:
+ 
+   `const auto client = UID2ClientFactory::Create(baseUrl, apiKey, secretKey);`
+2. Refresh once at startup, and then periodically (for example, every hour):
+
+   `client->Refresh();`
+3. Senders: 
+   1. Call the following:
+
+      `EncryptionResult encrypted = client->Encrypt(rawUid);`
+   2. If encryption succeeded, send the UID2 token to the receiver:   
+
+      `if (encrypted.IsSuccess()) { send encrypted.GetEncryptedData() to receiver} else {check encrypted.GetStatus()for the failure reason}` 
+4. Receivers: 
+   1. Call the following:
+
+      `DecryptionResult decrypted = client->Decrypt(uidToken);`
+   2. If decryption succeeded, use the raw UID2:
+    
+      `if (decrypted.IsSuccess()) {use decrypted.GetUid() } else {check decrypted.GetStatus() for the failure reason }`
```

Comment:
better formatting as Andrei suggested above

Reply:
@sunnywu need help with this. I started... but it's not exactly parallel. Hoping for a short meeting to resolve this and possible other technical queries on these docs.